lib/std/pointers.nim

```diff
@@ -0,0 +1,15 @@
+##[
+Convenience procs to deal with pointer-like variables.
+]##
+
+proc toUncheckedArray*[T](a: ptr[T]): ptr UncheckedArray[T] {.inline.} =
+  ## calls `cast[ptr UncheckedArray[T]]` (less error prone).
```

Comment:
And this doc is just horrible... You should elaborate more and please put more effort in documentation. I think we have several "shortcut" procs in system.nim and maybe you can draw from those to make the documentation for this a bit more descriptive.

Currently this documentation make no attempt to explain what would casting to `UncheckedArray` achieve, and it does not warn that this is an unsafe operation (`UncheckedArray` is unsafe).

Reply:
fixed